.ci/test.sh

```diff
@@ -50,8 +50,10 @@ if [[ $TRAVIS == "true" ]] && [[ $TASK == "check-docs" ]]; then
     exit 0
 fi
 
-if [[ $TASK == "pylint" ]]; then
+if [[ $TASK == "lint" ]]; then
     conda install -q -y -n $CONDA_ENV pycodestyle pydocstyle
+    pip install --user cpplint
+    cpplint --filter=-build/header_guard,-whitespace/line_length --recursive ./src ./include || exit 0  # TODO: change to -1 after fixing #1990
```

Comment:
@StrikerRUS I'm a fan of never putting `TODO`s in code, but instead keeping such details in the issues. In this case, you could just add that comment to #1990 if you haven't already.

Also I'm a fan of having commands with more than one argument indented like below, so their git blames are cleaner and so that it's more obvious what they are doing:

```bash
cpplint \
    --filter=-build/header_guard,-whitespace/line_length \
    --recursive \
    ./src \
    ./include \
|| exit 0
```

Up to you on both of these!

Reply:
@jameslamb Thanks for your review!
Agree with your first comment. Will remove `TODO` from the code.

Also, agree with your second comment, but I'm a fan of having consistent state, even if it's not perfect 😃 . I mean, we have a lot of such commands with plenty of args in this (and neighboring) file and this line will be "rara avis" here. So, I think such style improvements should be done at once in a separate PR instead of iterative one-line corrections.